command/server/config.go

```diff
@@ -745,7 +745,7 @@ func ParseConfig(d, source string) (*Config, error) {
 	}
 
 	if err := result.parseConfig(list); err != nil {
-		return nil, fmt.Errorf("error parsing enterprise config: %w", err)
```

Comment:
I think both `command/server/config_util.go` and `internalshared/configutil/config_util.go` could be removed. :-) Will cause some build failures that you'll need to address.

Reply:
Only saw this comment today. Didn't think anyone would look into the PR while in draft.

Reply:
Do you think makes sense to also try to exclude all license related code in this PR?

Reply:
Do you think makes sense to also try to exclude all license related code in this PR?

Reply:
UPDATE: Looks like the build is now failing after removing the licensing references. Still not sure why but would like to understand. :thinking:

Reply:
Yeah, I think removing the licensing references is fair game. :-) I expected that to break the build... It looks like you might just have a few more references to licensing running around still. :-)

Reply:
Yes, there we still some licensing references around but I believe to have removed them all now. The problem seems to be in the `core` `systemBarrierView` which is nil.